src/router.ts

```diff
@@ -2,17 +2,18 @@ import { createRouter, createWebHashHistory } from 'vue-router';
 import domains from '@/../snapshot-spaces/spaces/domains.json';
 import aliases from '@/../snapshot-spaces/spaces/aliases.json';
 import Home from '@/views/Home.vue';
-import Proposal from '@/views/Proposal.vue';
-import Create from '@/views/Create.vue';
+import SpaceProposal from '@/views/SpaceProposal.vue';
```

Comment:
Can you make a folder /src/views/Space/ for thess routes?

Reply:
I'd rather slowly adopt a flat folder structure, as it's considered best practice for large scale vuejs apps.

You can find it at "A Flat Component Directory"
https://vueschool.io/articles/vuejs-tutorials/how-to-structure-a-large-scale-vue-js-application/

Also learned about this on FrontendMasters course.

Reply:
Actually reading that again I see some more improvements I can do on this PR